### PR TITLE
Http Client 5

### PR DESCRIPTION
**Pull request by kevinmcgoldrick**

### CODE REVIEW

File: agent/http_client_5/src/test/java/com/intuit/tank/httpclient5/MockLogUtil.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/**
```
Comment (kmarquardsen) [inline agent/http_client_5/src/test/java/com/intuit/tank/httpclient5/MockLogUtil.java]: remove unnecessary comments

File: agent/http_client_5/src/test/java/com/intuit/tank/httpclient5/MockLogUtil.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import java.util.Map;
+
+import com.intuit.tank.http.TankHttpLogger;
+import com.intuit.tank.logging.LogEventType;
+import com.intuit.tank.logging.LoggingProfile;
+import com.intuit.tank.vm.settings.AgentConfig;
+import com.intuit.tank.vm.settings.TankConfig;
+
+/**
+ * @author denisa
```
Comment (kmarquardsen) [inline agent/http_client_5/src/test/java/com/intuit/tank/httpclient5/MockLogUtil.java]: remove comments... IMHO I'd also prefer never adding @author tags as well. it's extraneous lines of code that feel possessive and provide no real value.

**Comment by kevinmcgoldrick** [comment]
1. No, not yet.  We would need to be completely on 4.5 before removing 3.
2. No, just settings.xml file changes (I believe, not tested).
3. never going to. Just copied along with the folder. :)

**Comment by kmarquardsen** [comment]
cool, should we remove the mock classes until we run into a situation where we'll be writing tests? don't want confusion regarding unused classes

**Comment by kmarquardsen** [comment]
need to rebase and resolve the conflicts before this can be merged

**Pull request merged**